flink-runtime/src/main/java/org/apache/flink/runtime/io/network/NetworkEnvironment.java

```diff
@@ -275,15 +276,15 @@ private void registerInputMetrics(MetricGroup inputGroup, MetricGroup buffersGro
 		buffersGroup.gauge(METRIC_INPUT_POOL_USAGE, new InputBufferPoolUsageGauge(inputGates));
 	}
 
-	public void start() throws IOException {
+	public int start() throws IOException {
```

Comment:
Let's have a doc comment for this method in particular about the return value, also in the `ConnectionManager` interface